tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -219,6 +219,11 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-tgl-nocodec-crossover.bin,EFX_CROSSOVE
 "development/cavs-nocodec-rtcaec\;sof-tgl-nocodec-rtcaec\;PLATFORM=tgl,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-tgl-nocodec-rtcaec.bin"
 
+# Add MFCC to 16 kHz DMIC1
+"sof-hda-generic\;sof-hda-generic-cavs25-2ch-mfcc\;HDA_CONFIG=mix,\
```

Comment:
Note to myself: Need to add generic 2ch and 4ch topologies. I made this 2ch because MFCC was configured by blob to select channel 1 from (16 kHz 16 bit) stereo.

Reply:
Pls, just add to inline comments as reminder.